genestack-application-manager

```diff
@@ -438,6 +461,25 @@ def show_info(files, vendor_only, with_filename, no_filename):
         first_file = False
 
 
+def prompt_removing_stable_version(application, apps_ids, version):
+    apps = []
```

Comment:
You can wrap function to `try/except KeyboardInterrupt` block and return `False`   
In that case you will print exit message if CTRL+C pressed